src/main/java/no/nav/veilarboppfolging/service/SisteEndringPaaOppfolgingBrukerService.java

```diff
@@ -0,0 +1,32 @@
+package no.nav.veilarboppfolging.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.common.types.identer.Fnr;
+import no.nav.veilarboppfolging.repository.SisteEndringPaaOppfolgingBrukerRepository;
+import org.springframework.stereotype.Service;
+
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class SisteEndringPaaOppfolgingBrukerService {
```

Comment:
Ref. kommentar om tester for `SisteEndringPaaOppfolgingBrukerRepository`, kan evt lage noen tester av denne service'en, og teste repository via denne.